class-00/lab-a/arrow-functions/app.js

```diff
@@ -108,42 +108,53 @@ const newObject = array => ({
 
 
 
-let sum = function(a, b, c, d) {
-  return a + b + c + d;
-};
+// let sum = function(a, b, c, d) {
+//   return a + b + c + d;
+// };
+
+let sum = (a,b,c,d) => `${a + b + c + d}`
```

Comment:
You don't need the `${}` in this function. By using those you are returning a string, not a number.